README.md

```diff
@@ -11,9 +11,9 @@
 npm install readable-stream
 ```
 
-This package is a mirror of the streams implementations in Node.js 18.19.0.
```

Comment:
Okay, so turns out we were on Node 18.19.0 before, not 18.20.5. 😅 

We should definitely consider updating, but maybe keep that for a separate PR. 😉

Reply:
Done - there appears to still be a larger diff than I would expect; seemingly due to eslint using a different version as a result of a lack of lockfile